lib/unifile-sftp.js

```diff
@@ -27,6 +27,10 @@ function parseError(err) {
 	throw error;
 }
 
+function protectPath(path) {
+	return path.length ? path : '/';
```

Comment:
Shouldn't it throw an error? Is it consistent with the other services?

Reply:
It was already throwing an error. But we had this kind of behavior in [Dropox](https://github.com/silexlabs/unifile/blob/master/lib/unifile-dropbox.js#L150) or [GitHub](https://github.com/silexlabs/unifile/blob/master/lib/unifile-github.js#L25) (but in the other way)